buildtest/cli/buildspec.py

```diff
@@ -957,6 +957,28 @@ def print_format_fields():
         table.add_row("type", "Display 'type' property in test")
         console.print(table)
 
+    @staticmethod
+    def print_raw_filter_fields():
+        """This method prints the raw filter fields available for buildspec cache. This
+        method implements command ``buildtest buildspec find --filterfields``
+        """
+        console.print("buildspecs")
+        console.print("executor")
+        console.print("tags")
+        console.print("type")
```

Comment:
you should not be printing this manually, please use the builtin list in the class its defined at top something like this


```
    filter_fields = ["type", "executor", "tags", "buildspec"]
    default_format_fields = ["name", "type", "executor", "tags", "description"]
    format_fields = default_format_fields + ["buildspec"]
```

Reply:
Done! @shahzebsiddiqui